syntax = "proto3";

package Proto;

message TestMsg {
    string msg = 1;
    int32 index = 2;
}

message AccountCheck {
    string account = 1;
    string password = 2;
}

message AccountCheckToHttpRs {
    int32 return_code = 1;
    string account = 2;
}

message AccountCheckRs {
    enum ReturnCode {
        ARC_OK = 0;
        ARC_UNKNOWN = 1;
        ARC_NOT_FOUND_ACCOUNT = 2;
        ARC_PASSWORD_WRONG = 3;
    }
    int32 return_code = 1;
}